src/EnergyPlus/BranchInputManager.cc

```diff
@@ -1063,12 +1062,6 @@ namespace BranchInputManager {
                 cNumericFields.deallocate();
                 lAlphaBlanks.deallocate();
                 lNumericBlanks.deallocate();
-                if (ErrFound) {
-                    ShowSevereError(
-                        state,
-                        format("{} Invalid {} Input, preceding condition(s) will likely cause termination.", RoutineName, CurrentModuleObject));
-                    state.dataBranchInputManager->InvalidBranchDefinitions = true;
-                }
                 TestInletOutletNodes(state);
```

Comment:
[src/EnergyPlus/BranchInputManager.cc:1066]:(style),[knownConditionTrueFalse],Condition 'ErrFound' is always false